lib/omise/OmiseCapabilities.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+class OmiseCapabilities extends OmiseApiResource
+{
+    const ENDPOINT = 'capability';
+
+    /**
+     * Retrieves capabilities.
+     *
+     * @param  string $publickey
+     * @param  string $secretkey
+     *
+     * @return OmiseCapabilities
+     */
+    public static function retrieve($publickey = null, $secretkey = null)
+    {
+        return parent::g_retrieve(get_class(), self::getUrl(), $publickey, $secretkey);
+    }
+
+    /**
+     * Retrieves array of payment backends. Optionally pass in as many filter functions as you want
+     *
+     * @param [function,...]
+     *    
+     * @return array
+     */
+    public function getBackends() {
```

Comment:
Still not convinced about naming convetion... get Backends..

Reply:
Don't blame me - it's what they're called in the Capability API response (and probably elsewhere) - I'm just following the naming convention